### ISSUE TITLE
High IO load in OpenVZ VPS

### ISSUE THREAD

**Issue by jtrott** [Radarr/Radarr#2327]
Running Radarr results in load going over 2 and staying there on my OpenVZ based VPS.

After the initial RSS scan has finished, Radarr is showing no activity in the logs and in the files logs but is chewing approx 2000K/sec in IO bandwidth on the VPS (as see in iotop).
Running Radarr under strace shows the following loop that it gets stuck in:

futex(0x99f1a4, FUTEX_WAIT_BITSET_PRIVATE|FUTEX_CLOCK_REALTIME, 2054, {1512774595, 312156000}, ffffffff) = -1 ETIMEDOUT (Connection timed out)
clock_gettime(CLOCK_MONOTONIC, {59373, 358978053}) = 0
futex(0x99f1e0, FUTEX_WAKE_PRIVATE, 1)  = 0
clock_gettime(CLOCK_MONOTONIC, {59373, 359051734}) = 0
clock_gettime(CLOCK_MONOTONIC, {59373, 359076468}) = 0

Those lines repeat endlessly and the IO continues without end.

Radarr 0.2.0.870
Mono JIT compiler version 4.6.2 (Stable 4.6.2.16/ac9e222 Mon Jul 31 05:33:23 UTC 2017)
CentOS Linux release 7.4.1708 (Core)

**Comment by galli-leo** [comment]
Can you enable trace logging and see if there is really no activity?

**Comment by jtrott** [comment]
I've tracked it down to Radarr renaming my movie files - even though that has been disabled in the settings.
RenameMovieFileService keeps triggering on a ?daily? schedule for some reason.
I have a unison mount that has a read only cloud drive overlaid on top of a copy-on-write local storage to make It easier for Sonarr and Radarr to write to the cloud. I then have a script that uploads the local storage data to the cloud, then deletes the local storage data.
I have now switched my post processing to filebot for various reasons, so to mitigate this issue I have now mounted my unison as read only, so Radarr can't trigger massive data write by renaming files even when I've told it not to.
